.circleci/config.yml

```diff
@@ -44,6 +44,10 @@ jobs:
           command: |
             ./project.rb circle-deploy \
               --project all-of-us-workbench-test --creds-file circle-sa-key.json
+      - run:
+          name: Scan dependencies for vulnerabilities
+          working_directory: ~/workbench/api
+          command: ./project.rb gradle dependencyCheckAnalyze --info
```

Comment:
Will this cause the build to fail if there's a security vulnerability? (I assume that's what we want?)

Reply:
The build would always fail since there are a bunch right now. I talked this over with @jmandel and we decided to report without failing in this initial version.
Unrelated problem: The command exited abnormally and that SHOULD have failed the build. Added a fix for that.